tools/styles/fonts.css

```diff
@@ -1,6 +1,5 @@
-
 /*
-Copyright 2019 Adobe. All rights reserved.
+Copyright 2023 Adobe. All rights reserved.
```

Comment:
This looks like it's the same as `tools/styles/typography.css` now. I _think_ we can remove this and short up any internal usage.